nexmark-flink/src/main/resources/queries-cep/q1.sql

```diff
@@ -0,0 +1,33 @@
+-- -------------------------------------------------------------------------------------------------
+-- Query 1: Sum of the prices mapped to A and overall match (Not in original suite)
+-- -------------------------------------------------------------------------------------------------
+-- Returns the sum of the prices of all rows mapped to A and the first and last timestamp of the overall match.
+-- Illustrates a AFTER MATCH strategy SKIP PAST LAST ROW in MATCH_RECOGNIZE.
+-- -------------------------------------------------------------------------------------------------
+
+CREATE TABLE discard_sink (
+      bidder  BIGINT,
+      sumPrice  BIGINT,
+      startTime  TIMESTAMP(3),
+      endTime  TIMESTAMP(3)
+) WITH (
+    'connector' = 'blackhole'
+);
+
+INSERT INTO discard_sink
+SELECT
+    bidder, sumPrice, startTime, endTime
+FROM bid
+MATCH_RECOGNIZE(
+    PARTITION BY bidder
+    ORDER BY dateTime
+    MEASURES
+        SUM(A.price) AS sumPrice,
+        FIRST(dateTime) AS startTime,
+        LAST(dateTime) AS endTime
+    ONE ROW PER MATCH
+    AFTER MATCH SKIP PAST LAST ROW
+    PATTERN (A+ C)
+    DEFINE
+        A AS SUM(A.price) < 10000
```

Comment:
Personally I prefer to change it to something like `COUNT(A.price) < 5` for the following reasons:
- For `SUM(A.price)`, it means that we need to carefully define the value scope of random generated price. If it's too large, this means that it will always match the first input. Otherwise, if it's too small, it means that this condition never holds. 
- This is bid price for the same item. What's the semantic of summarizing them?